docs/www/support.md

```diff
@@ -165,7 +165,7 @@ misconfigurations.
 ### The 'redpanda' package wasn't found
 
 Make sure that you added our repo to your package manager as explained in our
-[Quick Start Linux Guide](/docs/quick-start-linux)
+[Quick Start Linux Guide](quick-start-linux.md)
```

Comment:
i don't see how this makes sense tbh. why is removing /docs/ make sense at all? this is deployed to `vectorized.io/docs` so we need the `/docs` prefix.

Reply:
I see. I was looking at it only from a github reader perspective